# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: toolkit/content/xul.css
# Commit: 0a3da2090e48
# Full Hash: 0a3da2090e4859473f52b585bcb39baae2876b28
# Author: Jared Wein <jwein@mozilla.com>
# Date: 2021-02-26 03:55:47
# Regressor Bug: 1665390
# File Overlap Count: 1
# Description:
#   Bug 1665390 - Style bookmarks toolbar and tabs toolbar tooltips. r=NeilDeakin,ntim
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D104257
# ==============================================================================

diff -r 176b788753e8 -r 0a3da2090e48 toolkit/content/xul.css
--- a/toolkit/content/xul.css	Thu Feb 25 20:47:15 2021 +0000
+++ b/toolkit/content/xul.css	Thu Feb 25 20:47:54 2021 +0000
@@ -262,6 +262,9 @@
 tooltip {
   -moz-box-orient: vertical;
   white-space: pre-wrap;
+}
+
+tooltip:not([position]) {
   margin-top: 21px;
 }
 